iterations_runners/iterations_runner.java

```diff
@@ -181,20 +184,20 @@ public static void main(String args[]) throws
          *   - We print the result using toUnsignedString, thus interpreting
          *     the number as unsigned.
          */
-        long[] tsr_iter_times = new long[iterations];
-        Arrays.fill(tsr_iter_times, 0);
+        long[] cycleCounts = new long[iterations];
+        Arrays.fill(cycleCounts, 0);
 
         // Please, no reflection inside the timed code!
```

Comment:
hm, this is just an unrelated observation, but we should remove that comment, we haven't been using reflection since ages. Relatedly, we can probably remove some of the exceptions from the declaration of main. Should I make an issue so we don't forget?